slidez-core/src/main/java/com/binarystudio/academy/slidez/app/SessionController.java

```diff
@@ -0,0 +1,72 @@
+package com.binarystudio.academy.slidez.app;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import com.binarystudio.academy.slidez.domain.session.SessionService;
+import com.binarystudio.academy.slidez.domain.session.dto.SessionUpdateDto;
+import com.binarystudio.academy.slidez.domain.session.model.Session;
+import com.binarystudio.academy.slidez.domain.user.dto.UserDetailsDto;
+import com.binarystudio.academy.slidez.domain.user.mapper.UserMapper;
+import com.binarystudio.academy.slidez.domain.user.model.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("${v1API}/sessions")
+public class SessionController {
+
+	@Autowired
+	SessionService sessionService;
+
+	@PostMapping("/create")
+	@ResponseStatus(HttpStatus.CREATED)
+	public UUID save(@RequestBody Session session) {
+		Session createdSesion = sessionService.create(session);
+		UUID id = createdSesion.getId();
+		return id;
+	}
+
+	@PostMapping("/update")
```

Comment:
Isn't it better to use PUT for update?

Reply:
POST means "create new". PUT means "insert, replace if already exists".